src/main/java/vazkii/quark/content/world/module/BlossomTreesModule.java

```diff
@@ -47,8 +47,12 @@ public void construct() {
 	@Override
 	public void setup() {
 		trees.forEach((tree, config) -> {
-			ComposterBlock.CHANCES.put(tree.leaf.getBlock().asItem(), 0.3F);
-			ComposterBlock.CHANCES.put(tree.sapling.asItem(), 0.3F);
+			if (tree.leaf.getBlock().asItem() != null) {
+				ComposterBlock.CHANCES.put(tree.leaf.getBlock().asItem(), 0.3F);
+			}
+			if (tree.sapling.asItem() != null) {
+				ComposterBlock.CHANCES.put(tree.sapling.asItem(), 0.3F);
+			}
```

Comment:
Please try and match the current formatting. One-line statements should not have braces.